projects/igniteui-angular/src/lib/data-operations/filtering-condition.ts

```diff
@@ -890,6 +947,7 @@ export class IgxStringFilteringOperand extends IgxFilteringOperand {
 export interface IFilteringOperation {
     name: string;
     isUnary: boolean;
+    isNestedQuery: boolean;
```

Comment:
We should mark this optional, otherwise we will introduce a breaking change.

Reply:
Marked as optional